share/etest/runners.sh

```diff
@@ -108,6 +108,15 @@ run_single_test()
                 suite_setup
             fi
 
+            # Register __suite_teardown function as a trap callback and trigger it when the test receives an interrupt.
+            # If running the last test case, instead of triggering it by interrupt, make this function always triggered
+            # at the end of the test. 
```

Comment:
Nit: Reword this to

|  ... instead of triggering it by interrupt, always trigger it at the end of the test.

Reply:
fixed, thanks for the suggestion!